usecases/auth/authorization/rbac/manager.go

```diff
@@ -41,6 +41,10 @@ func New(rbacStoragePath string, rbac rbacconf.Config, logger logrus.FieldLogger
 
 func (m *manager) UpsertRolesPermissions(roles map[string][]authorization.Policy) error {
 	for roleName, policies := range roles {
+		// dumb name of the role without permissions to make sure it does exists even if there was no permissions
+		if _, err := m.casbin.AddRoleForUser(conv.PrefixUserName(InternalPlaceHolder), conv.PrefixRoleName(roleName)); err != nil {
```

Comment:
instead of using a random name, could we have another prefix for internal stuff?

Reply:
>that's not random name, that is a constant internal name, it has to be mapped in user<->roles relationship to be able to detect staled roles

random in the sense, that you choose a name and there is nothing special about it, eg no internal only prefix or something. What would happen if we had the user `wv_internal_empty`?

Reply:
alright, maybe we can rediscuss next cycle. Will resolve for now